pkg/oc/admin/image/verify-signature.go

```diff
@@ -123,6 +136,9 @@ func (o *VerifyImageSignatureOptions) Validate() error {
 	if o.RemoveAll && len(o.ExpectedIdentity) > 0 {
 		return errors.New("the --expected-identity cannot be used when removing all verifications")
 	}
+	if len(o.RegistryURL) == 0 && o.Insecure {
+		return errors.New("the --insecure can be used only with --registry-url")
```

Comment:
s/--insecure can be used only with/--insecure flag can only be used with/

Reply:
I believe the whole block can be deleted now. The flag should be honored for both routes.

Reply:
yeah, this is not needed anymore